torch/ao/quantization/qconfig.py

```diff
@@ -323,3 +331,15 @@ def partial_equals(p1, p2):
             return partial_equals(q1.activation.p, q2.activation.p) and partial_equals(q1.weight.p, q2.weight.p)
         except AttributeError:
             return q1 == q2
+
+def observer_is_memoryless(qconfig: QConfigAny):
```

Comment:
may want to change this function name, you're really checking if the qconfig activation makes a fake_quant and if that fake_quant has a memoryless observer or if the activation makes a memoryless observer.

i.e. maybe a better name would be `activation_is_memoryless`